### DISCUSSION TITLE
Import batch entries from csv file

### DISCUSSION THREAD

**Discussion by garatortiz** [strapi/strapi#6933] [Legacy Topics]
We have the following problem:
we need to import a batch of entries from a csv file, and we find two issues to solve:

- The first of them is that we have to upload to Strapi images that come from a URL in a field of the file.
- The second, and the one that we have been least able to solve, is that several of the fields correspond to Content-Types that must be nested, and of which in the csv file we only have the name. Any ideas or solutions to this?

**Comment by MattieBelt** [comment]
Hey @garatortiz, did you check out this community plugin? 
https://github.com/pouyamiralayi/strapi-import-content-plugin-tutorial
It could provide an solution to the second problem. (Not sure about that, I have no experience with it)
